### DISCUSSION TITLE
Top Player Online

### DISCUSSION THREAD

**Discussion by BrayaanPS** [Zrips/CMI#2906] [Feature Requests]
Hi, this is my suggestion.
It would not be good if there was a variable to create a top of the players more time connected to the server, just like the variables of top money and votes.
* Placeholder:  `%cmi_playeronline_[1-10]%`
It is variable what would make our servers very interesting (in my opinion)

**Comment by Zrips** [comment]
This can be easily abused by simply reloging in and out for dozen times in a row and would only incentivize people in doing so.

**Comment by BrayaanPS** [comment]
No, I get it wrong. a top of players with more hours connected.
